a/apache-arrow/arrow_ubi_9.3.sh

```diff
@@ -0,0 +1,122 @@
+
+wget "https://go.dev/dl/$GO_VERSION.linux-ppc64le.tar.gz"
+rm -rf /usr/local/go
+tar -C /usr/local -xf $GO_VERSION.linux-ppc64le.tar.gz
+export GOROOT=/usr/local/go
+export GOPATH=$HOME
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+
+
+go install gotest.tools/gotestsum@latest
+go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
+
+if [ -d "$PACKAGE_NAME" ] ; then
+rm -rf $PACKAGE_NAME
+echo "$PACKAGE_NAME | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+fi
+
+# Cloning the repository from remote to local
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+git submodule init
+git submodule update
+export PARQUET_TEST_DATA="${PWD}/cpp/submodules/parquet-testing/data"
+export ARROW_TEST_DATA="${PWD}/testing/data"
+
+go install honnef.co/go/tools/cmd/staticcheck@latest
+
+# Build Arrow C++ library
+mkdir cpp/build
+cd cpp/build
+cmake .. \
+  -DARROW_WITH_SNAPPY=ON \
+  -DARROW_WITH_ZLIB=ON \
+  -DARROW_WITH_LZ4=ON \
+  -DARROW_WITH_ZSTD=ON \
+  -DARROW_PARQUET=ON \
+  -DARROW_CSV=ON \
+  -DARROW_DATASET=ON \
+  -DARROW_BUILD_TESTS=OFF \
+  -DARROW_BUILD_UTILITIES=OFF \
+  -DARROW_BUILD_SHARED=ON \
+  -DARROW_PYTHON=ON \
+  -DPython3_EXECUTABLE=/usr/bin/python3 \
+  -DARROW_JEMALLOC=OFF \
+  -DCMAKE_BUILD_TYPE=Release
+make -j4
+sudo make install
+
+# Build Arrow Go bindings
+cd ../../go
+go mod tidy
+
+if ! go build -tags arrow -v ./... ; then
+    echo "------------------$PACKAGE_NAME:Build_fails---------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Build_Fails"
+    exit 1
+fi
+
+if ! go test -tags arrow -v ./... ; then
+    echo "------------------$PACKAGE_NAME::Build_and_Test_fails-------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Fail|  Build_and_Test_fails"
+    exit 2
+else
+    echo "------------------$PACKAGE_NAME::Build_and_Test_success-------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Build_and_Test_Success"
+fi
+
+
+# ==================== Build the Python Wheel (.whl) file =================================
+cd ../python
+# Install necessary Python build tools
+pip${PYTHON_VER} install --upgrade setuptools wheel numpy
+python${PYTHON_VER} -m pip install Cython==3.0.8
+python${PYTHON_VER} setup.py build_ext --inplace
+python${PYTHON_VER} setup.py bdist_wheel
+
+# Find and display the generated WHL file
+WHL_FILE=$(ls dist/*.whl)
+if [ -f "$WHL_FILE" ]; then
+    echo "------------------$PACKAGE_NAME::WHL file generated successfully-------------------------"
+    echo "The generated WHL file is located at: $WHL_FILE"
+    exit 0
+else
+    echo "------------------$PACKAGE_NAME::WHL file generation failed-------------------------"
+    exit 3
+fi
```

Comment:
ok

Reply:
@nikhil-kalbande  I don't think we should be having wheel creation step here. 
@puneetsharma21 you might need to add `export` statements for values that is going to use in building process. For example `BUILD_TYPE=release BUNDLE_ARROW_CPP=1`, as we are using `python -m build --wheel --outdir="$CURRENT_DIR/"` in [wrapper script](https://github.com/Vipul-Ajmera/build-scripts/blob/python_ecosystem/script/build_script_python_create_wheel.sh). 
@nikhil-kalbande @Vipul-Ajmera  Also as per current version of wrapper script. It will fallback with no wheel creation, because here we have to navigate to `python` dir and there the setup.py file is present. We had this case covered in our initial python-eco-system template of bulksearch